### DISCUSSION TITLE
Bullhorn updates December 14

### DISCUSSION THREAD

**Discussion by moosewebdev** [bullhorn/career-portal#501] [Q&A]
My client received a email notifying them of an update to redirect 
"
·         If your current REST API Key’s Redirect URI is set to [https://www.bullhorn.com](https://go.bullhorn.com/MTMxLVlRSy01NjgAAAGIaP47EHn3unDiH4roVDaBMKxXTKcphrtQqGlEXocl5do_Qj6nKo0OLOfpDnATW1_bkJSMGlI=), it will be updated to [https://welcome.bullhornstaffing.com](https://go.bullhorn.com/MTMxLVlRSy01NjgAAAGIaP47EG2f9yU17_uyv0tLQd0lXOK2HyOG7-YDyJHGlsnMnxshc-cit8YCvgwDxMbV-YmGn9c=)

·         If your API integration is enforcing or validating a redirect to [https://www.bullhorn.com](https://go.bullhorn.com/MTMxLVlRSy01NjgAAAGIaP47EHn3unDiH4roVDaBMKxXTKcphrtQqGlEXocl5do_Qj6nKo0OLOfpDnATW1_bkJSMGlI=)  during the OAuth process, please remove this logic prior to this maintenance to avoid errors. Developers utilizing the standard Bullhorn SDK should not be impacted by this change."

Do i need to be worried about this?

**Comment by pbearne** [comment]
No Action the OSCP is using a swimlane so I don't think this will affect it.

Matador Jobs has got loads of questions about this!!

Paul
